### PR TITLE
fix(payments) Basic payment submission error message update

### LINKED ISSUE #2914: (fxa-payments-server) Update general payment submission error message
Spin-off of #2812 until error messages there can be better sussed out by @johngruen.

The basic error message should read "Something went wrong. Please try again later."

### PR DESCRIPTION
**Pull request by LZoog**
fixes #2914

### CODE REVIEW

File: packages/fxa-payments-server/src/lib/errors.ts [hunk 1]
```diff
@@ -16,8 +16,7 @@ const AuthServerErrno = {
  * - handle Payment token not valid
  */
 
-const BASIC_ERROR =
-  "Hmm, we're having trouble with our system. We're working on fixing it for you and apologize for the inconvenience. Please try again later.";
```
Comment (lmorchard) [inline packages/fxa-payments-server/src/lib/errors.ts]: FWIW, this was a message reviewed & approved by legal. Might be fine to change, but may be an issue?

Comment (LZoog) [reply packages/fxa-payments-server/src/lib/errors.ts]: 👍 Thanks for this context. @johngruen sorry for so many @ mentions, but you think we should run this by anyone first?

Comment (johngruen) [reply packages/fxa-payments-server/src/lib/errors.ts]: yeah, getting legal sign off ASAP

Comment (johngruen) [reply packages/fxa-payments-server/src/lib/errors.ts]: Legal is cool

**Pull request merged**